app/models/persistent_volume.rb

```diff
@@ -3,4 +3,7 @@ class PersistentVolume < ContainerVolume
   include NewWithTypeStiMixin
   serialize :capacity, Hash
   delegate :name, :to => :parent, :prefix => true
+  has_many :container_volumes, :through => :persistent_volume_claim
```

Comment:
@zakiva PTAL at the [following tests](https://github.com/ManageIQ/manageiq-providers-kubernetes/issues/3) .